lms/static/sass/base/_mixins.scss

```diff
@@ -171,6 +171,16 @@
   text-overflow: ellipsis;
 }
 
+%vertical-aligner {
```

Comment:
Also, might be good to add a comment before this new placeholder to explain how/when its good to use it.